deploy-agent/deployd/common/executor.py

```diff
@@ -78,7 +78,7 @@ def run_cmd(self, cmd, **kw):
                     fdout.seek(0, 2)
                     file_pos = fdout.tell()
                     process = subprocess.Popen(cmd, stdout=fdout, stderr=fdout,
-                                               preexec_fn=os.setsid, **kw)
+                                               preexec_fn=os.setsid, env=self._config.get_all_environ_vars(), **kw)
```

Comment:
running into this problem
`  File "/root/.pex/installed_wheels/77e2eeedf477b943ce783c4f3806280d8147719c/deploy_agent-1.2.28-py2-none-any.whl/deployd/common/executor.py", line 81, in run_cmd
    preexec_fn=os.setsid, env=self._config.get_all_environ_vars(), **kw)
  File "/usr/lib/python2.7/subprocess.py", line 394, in __init__
    errread, errwrite)
  File "/usr/lib/python2.7/subprocess.py", line 1047, in _execute_child
    raise child_exception
OSError: [Errno 2] No such file or directory`